syntax = "proto3";

package response_models;

message UpdatePlayerAmmoResponseModel {
	int32 player_id = 1;
	int32 new_ammo = 2;
}